service/templates/confirm_selection.html

```diff
@@ -2,58 +2,133 @@
+            <p>Most flats are leasehold. Houses can be leasehold too and usually are if they’re bought through a shared ownership scheme.</p>
+            <p>The leasehold and freehold are registered separately with the Land Registry. Each has its own title documents. <b class="bold-xsmall">You are buying a summary of the leasehold owner, not the freehold.</b></p>
+
+            <a rel="external" href="https://www.gov.uk/leasehold-property/overview">More about Leasehold on GOV.UK</a>
+
+          {% elif params['title']['tenure'].upper() == 'FREEHOLD' %}
+            <p>You own a freehold building and land outright. It is yours until you sell it or give it away.</p>
+            <p>Most houses are freehold. Most flats are leasehold. That means someone else owns the freehold for the building the flat is in. <b class="bold-xsmall">If you want to know who owns a particular flat, you probably need the leasehold.</b></p>
+            <p>Houses can also be leasehold and usually are if they’re bought through a shared ownership scheme.</p>
+
+          {% endif %}
+          </div>
+        </details>
+
+        <p class="font-xsmall">If you require other types of tenure for this property, <a href="#" onClick="history.go(-1); return false;">go back</a> and select the one you need.</p>
+      </dd>
     </dl>
 
-    <br/>
+    <h2 class="heading-medium collapse-top">The summary will tell you</h2>
+    <ul class="list-bullet">
+      {% if params['title']['tenure'].upper() == 'FREEHOLD' %}
+        <li>current owner’s name and address</li>
+      {% elif params['title']['tenure'].upper() == 'LEASEHOLD' %}
+        <li>current leaseholder’s name and address</li>
+      {% endif %}
+      <li>what they paid for the property – If the whole property was sold on the open market, after April 2000. <span class="font-xsmall">(<a href="https://www.gov.uk/guidance/about-the-price-paid-data#data-excluded-from-the-house-price-index-and-price-paid-data" class="font-xsmall">what does this mean?</a>)</span></li>
+      <li>if there is a mortgage on the property, the lender’s name and  address</li>
+    </ul>
+    <!-- <p>You'll need to <a target="_blank" href="https://www.gov.uk/get-information-about-property-and-land">use a different Land Registry service</a> if you need more detailed information or to prove ownership in court.</p> -->
+    <div class="notice" style="background-color:#fff2d3;padding: 25px">
+      <img src="/static/govuk_frontend_toolkit/images/icon-important.png" alt="" style="float: left;margin-right: 15px;" />
+      <p style="overflow:hidden;margin:0;"><strong class="bold-small">You'll only be able to view the information once</strong><br>
+        If you want to keep a copy you can print it out</p>
+    </div>
+
+      <h2 class="heading-medium">The summary costs £2</h2>
```

Comment:
aha. This is why your tests work?
Should be using the variable { params.price }